scopes/workspace/workspace-config-files/ws-config.cmd.ts

```diff
@@ -124,9 +124,9 @@ export class WsConfigCleanCmd implements Command {
     [
       'w',
       'writers <writers>',
-      `only write config files for the given writers. use comma to separate multiple writers. use ${COMMAND_NAME} list to see all writers`,
+      `only clean config files for the given writers. use comma to separate multiple writers. use ${COMMAND_NAME} list to see all writers`,
     ],
-    ['', 'dry-run', 'show the paths that configs will be written per env'],
+    ['', 'dry-run', 'show the paths of configs that will be cleaned'],
```

Comment:
dry-run is for both: clean and write

Reply:
right, but here it's a flag on the clean command, so it's relevant only to dry-run what will be cleaned